lib/oli_web/live/sections/enrollments/invalid_section_invite_view.ex

```diff
@@ -7,8 +7,8 @@ defmodule OliWeb.Sections.InvalidSectionInviteView do
 
   def render(assigns) do
     ~H"""
-    <div>
-      This section invitation link has expired or is invalid.
+    <div class="p-10">
+      This enrollment link has expired or is invalid. If you already have a Torus student account, please <a href="/session/new">sign in</a>.
```

Comment:
We shouldn't mention "Torus" by name anywhere as it limits the ease of rebranding.  

This should instead simply be `If you already have a student account...`